app/src/main/java/com/google/samples/apps/sunflower/compose/garden/GardenScreen.kt

```diff
@@ -0,0 +1,265 @@
+
+    // Dimensions
+    val cardSideMargin = dimensionResource(id = R.dimen.card_side_margin)
+    val marginNormal = dimensionResource(id = R.dimen.margin_normal)
+
+    Card(
+        onClick = { onPlantClick(plant) },
+        modifier = Modifier.padding(
+            start = cardSideMargin,
+            end = cardSideMargin,
+            bottom = dimensionResource(id = R.dimen.card_bottom_margin)
+        ),
+        elevation = dimensionResource(id = R.dimen.card_elevation),
+        shape = MaterialTheme.shapes.card,
+    ) {
+        Column(Modifier.fillMaxWidth()) {
+            SunflowerImage(
+                model = vm.imageUrl,
+                contentDescription = plant.plant.description,
+                Modifier
+                    .fillMaxWidth()
+                    .height(dimensionResource(id = R.dimen.plant_item_image_height)),
+                contentScale = ContentScale.Crop,
+            )
+
+            // Plant name
+            Text(
+                text = vm.plantName,
+                Modifier
+                    .padding(vertical = marginNormal)
+                    .align(Alignment.CenterHorizontally),
+                style = MaterialTheme.typography.subtitle1,
+            )
+
+            // Planted date
+            Text(
+                text = stringResource(id = R.string.plant_date_header),
+                Modifier.align(Alignment.CenterHorizontally),
+                fontWeight = FontWeight.Bold,
+                color = MaterialTheme.colors.primaryVariant,
+                style = MaterialTheme.typography.subtitle2
+            )
+            Text(
+                text = vm.plantDateString,
+                Modifier.align(Alignment.CenterHorizontally),
+                style = MaterialTheme.typography.subtitle2
+            )
+
+            // Last Watered
+            Text(
+                text = stringResource(id = R.string.watered_date_header),
+                Modifier
+                    .align(Alignment.CenterHorizontally)
+                    .padding(top = marginNormal),
+                fontWeight = FontWeight.Bold,
+                color = MaterialTheme.colors.primaryVariant,
+                style = MaterialTheme.typography.subtitle2
+            )
+            Text(
+                text = vm.waterDateString,
+                Modifier.align(Alignment.CenterHorizontally),
+                style = MaterialTheme.typography.subtitle2
+            )
+            Text(
+                text = pluralStringResource(
+                    id = R.plurals.watering_next,
+                    count = vm.wateringInterval,
+                    vm.wateringInterval
+                ),
+                Modifier
+                    .align(Alignment.CenterHorizontally)
+                    .padding(bottom = marginNormal),
+                style = MaterialTheme.typography.subtitle2
+            )
+        }
+    }
+}
+
+@Composable
+private fun EmptyGarden(onAddPlantClick: () -> Unit) {
+    Column(
```

Comment:
You missed the report drawn here.
Depending on when you check the issue, today is scheduled the alpha03 which contains `ReportDrawn()`, so you may use it instead of the `ReportDrawnWhen { true }`

```suggestion
    ReportDrawnWhen { true}
    Column(
```

Reply:
Thanks - I wasn't sure if `ReportDrawnWhen { true }` is needed in this case since it's always `true`.